docs/book/_config.yml

```diff
@@ -23,4 +23,4 @@ html:
 sphinx:
   config:
     html_js_files:
-    - https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js
+      - https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js
```

Comment:
@MaxGhenis I know this seems like a syntax fix but the UK config has the old version, so might be either unnecessary or could be the reason for Plotly's failure?

Reply:
Yeah I'll try the UK config. This is my autoformatter btw, filed https://github.com/PolicyEngine/openfisca-us/issues/488 to look into this, ideally we'd run a formatter on all files once and no longer have these pop up.